src/lib/integration_tests/tests/platform_orchestrator/reset_ml_feed_cache_test.rs

```diff
@@ -0,0 +1,385 @@
+        )
+        .map(|reply_payload| {
+            let subnet_capacity: u64 = match reply_payload {
+                WasmResult::Reply(payload) => candid::decode_one(&payload).unwrap(),
+                _ => panic!("\n🛑 get_subnet_backup_capacity failed\n"),
+            };
+            subnet_capacity
+        })
+        .unwrap();
+    println!("Backup capacity: {:?}", res);
+
+    // create user canisters
+
+    let alice_individual_template_canister_id = pocket_ic
+        .update_call(
+            user_index_canister_id,
+            alice_principal_id,
+            "get_requester_principals_canister_id_create_if_not_exists",
+            encode_one(()).unwrap(),
+        )
+        .map(|reply_payload| {
+            let result: Result<Principal, String> = match reply_payload {
+                WasmResult::Reply(payload) => candid::decode_one(&payload).unwrap(),
+                _ => panic!(
+                    "\n🛑 get_requester_principals_canister_id_create_if_not_exists failed\n"
+                ),
+            };
+            result
+        })
+        .unwrap()
+        .unwrap();
+    println!("res1: {:?}", alice_individual_template_canister_id);
+
+    let bob_individual_template_canister_id = pocket_ic
+        .update_call(
+            user_index_canister_id,
+            bob_principal_id,
+            "get_requester_principals_canister_id_create_if_not_exists",
+            encode_one(()).unwrap(),
+        )
+        .map(|reply_payload| {
+            let result: Result<Principal, String> = match reply_payload {
+                WasmResult::Reply(payload) => candid::decode_one(&payload).unwrap(),
+                _ => panic!(
+                    "\n🛑 get_requester_principals_canister_id_create_if_not_exists failed\n"
+                ),
+            };
+            result
+        })
+        .unwrap()
+        .unwrap();
+    println!("res2: {:?}", bob_individual_template_canister_id);
+
+    let dan_individual_template_canister_id = pocket_ic
+        .update_call(
+            user_index_canister_id,
+            dan_principal_id,
+            "get_requester_principals_canister_id_create_if_not_exists",
+            encode_one(()).unwrap(),
+        )
+        .map(|reply_payload| {
+            let result: Result<Principal, String> = match reply_payload {
+                WasmResult::Reply(payload) => candid::decode_one(&payload).unwrap(),
+                _ => panic!(
+                    "\n🛑 get_requester_principals_canister_id_create_if_not_exists failed\n"
+                ),
+            };
+            result
+        })
+        .unwrap()
+        .unwrap();
+    println!("res3: {:?}", dan_individual_template_canister_id);
+
+    pocket_ic.add_cycles(alice_individual_template_canister_id, 2_000_000_000_000);
+    pocket_ic.add_cycles(bob_individual_template_canister_id, 2_000_000_000_000);
+    pocket_ic.add_cycles(dan_individual_template_canister_id, 2_000_000_000_000);
+
```

Comment:
Individual canisters asks for cycles when required. Is it failing without these lines? if so it will run of cycles on mainnet.

Reply:
no specific reason. simply copied from a previous test. will remove and check